editions/tw5.com/tiddlers/system/doc-macros.tid

```diff
@@ -107,3 +107,23 @@ This is an example tiddler. See [[Table-of-Contents Macros (Examples)]].
 </$reveal>
 </$list>
 \end
+
+\define demo(text)
```

Comment:
Hi @tobibeer were you not able to reuse the macros we've already got in `$:/editions/tw5.com/wikitext-macros`?